cmake/setupFFTW.cmake

```diff
@@ -0,0 +1,65 @@
+CPMAddPackage(
+    NAME findfftw
+    GIT_REPOSITORY "https://github.com/egpbos/findFFTW.git"
+    GIT_TAG "master"
+    EXCLUDE_FROM_ALL YES
+    GIT_SHALLOW YES
+)
+
+list(APPEND CMAKE_MODULE_PATH "${findfftw_SOURCE_DIR}")
+
+if (FINUFFT_FFTW_LIBRARIES STREQUAL DEFAULT OR FINUFFT_FFTW_LIBRARIES STREQUAL DOWNLOAD)
+    find_package(FFTW)
+    if ( (NOT FFTW_FOUND ) OR (FINUFFT_FFTW_LIBRARIES STREQUAL DOWNLOAD))
```

Comment:
I guess the if in line 13 matches line 62 and line 64. While the indents in between line 14 and line 62 may be misleading.

Reply:
@DiamonDinoia I guess this is the new commits.

Reply:
The change has been reverted.